File: Backend/src/main/java/com/luckyseven/backend/domain/expense/dto/CreateExpenseResponse.java
package com.luckyseven.backend.domain.expense.dto;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import lombok.Builder;

@Builder
public record CreateExpenseResponse(
    Long id,
    BigDecimal amount,
    BigDecimal foreignBalance,
    BigDecimal balance,
    LocalDateTime createdAt,
    LocalDateTime updatedAt
) {

}

File: Backend/src/main/java/com/luckyseven/backend/domain/expense/dto/CreateExpenseResponse.kt
package com.luckyseven.backend.domain.expense.dto

import java.math.BigDecimal
import java.time.LocalDateTime

data class CreateExpenseResponse(
    val id: Long?,
    val amount: BigDecimal,
    val foreignBalance: BigDecimal?,
    val balance: BigDecimal?,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime
)
